packages/big-design/src/components/Alert/styled.ts

```diff
@@ -0,0 +1,79 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css, DefaultTheme, StyledComponent } from 'styled-components';
+
+import { getBorderStyle } from '../../utils';
+import { StyleableButton } from '../Button/private';
+import { Grid } from '../Grid';
+import { Link } from '../Link';
+import { TextProps } from '../Typography';
+import { StyleableH4, StyleableSmall } from '../Typography/private';
+
+import { AlertProps } from './Alert';
+
+export const StyledAlert = styled(Grid)<AlertProps>`
+  ${({ theme }) => theme.shadow.floating}
+
+  animation: ${({ theme }) => theme.keyframes.fadeIn} .5s ease-in-out;
+  background-color: ${({ theme }) => theme.colors.white};
+  grid-gap: ${({ theme }) => theme.spacing.small};
+  max-width: ${({ theme }) => theme.helpers.remCalc(456)};
+  padding: ${({ theme }) => theme.spacing.small};
+  position: fixed;
+  right: ${({ theme }) => theme.helpers.remCalc(16)};
```

Comment:
Is there a reason we aren't using `spacing.medium` for these?

Reply:
Good catch.